### DISCUSSION TITLE
How to update phy? I noticed that there is interface ”add_connection_phy_changed“ in bleak_winrt, but I don't know how to use it correctly

### DISCUSSION THREAD

**Discussion by CheerMan1** [hbldh/bleak#855] [Q&A]
* bleak version:
* Python version:
* Operating System:
* BlueZ version (`bluetoothctl -v`) in case of Linux:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

It is preferable if an issue contains a [Miminal Workable Example](https://stackoverflow.com/help/minimal-reproducible-example).
This will otherwise be one of the first questions you will get as a response.

It is also preferable if that example is not dependent on a specific peripheral device, but can be run and
reproduced with other BLE peripherals as well.

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here as well.
```

**Comment by CheerMan1** [comment]
thank you for your reply

**Comment by CheerMan1** [comment]
Can you provide an example of updating the MTU and PHY? Thank you very much

**Comment by dlech** [comment]
Most of the operating systems supported by Bleak do not have APIs to set either the MTU or PHY so this functionality is not available.

**Reply by CheerMan1** [reply]
> Most of the operating systems supported by Bleak do not have APIs to set either the MTU or PHY so this functionality is not available.

Thank you very much for your reply and you are a great engineer
So do we have another way to update the PHY and MTU using the Python API?

**Reply by dlech** [reply]
No, this is all done automatically by the OS.
